build.sh

```diff
@@ -82,7 +83,7 @@ fi
 if [ "$test" == "true" ]; then
 	echo '::group::Building package with makepkg'
 	cd /tmp/local-repo/
-	makepkg --clean --cleanbuild --nodeps
+	makepkg $test_flags
```

Comment:
I don't want to see `shellcheck`'s complains.

```suggestion
	eval makepkg "$test_flags"
```

If you have a better way to explicitly specify that `$test_flags` is an array, feel free to implement them.